components/BoundaryLayer.svelte

```diff
@@ -1,11 +1,11 @@
-<script>
+<script lang="ts">
   import geojsonExtent from "@mapbox/geojson-extent";
   import mask from "@turf/mask";
   import {
     drawPolygon,
     overwriteSource,
     overwriteLayer,
-  } from "../maplibre_helpers.js";
+  } from "../maplibre_helpers";
```

Comment:
This is pretty bizarre to me. Actually we could leave it as `.js`, even though we've renamed the file to `.ts`! There's also some tsconfig option we can provide to allow specifying `.ts` explicitly. But AFAICT, it's common practice to just leave off the file extension. Happy to do otherwise; guidance here was unclear and depended on the build process and stuff